android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -159,6 +159,15 @@ public class Utils {
 
     private static final int CLEAR_CLIPBOARD_INTERVAL = 60000; // In milliseconds
 
+    public static final List<String> KNOWN_TEST_CHAIN_IDS = Arrays.asList(
```

Comment:
I think these lists are better added in `android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java`?